collections/_posts/2022-07-01-Complex-Convolutional-Networks.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: review
+title: "Complex Convolutional Neural Networks for Image Reconstruction from IQ Signal"
+tags: complex-CNNs IQ-signal image-reconstruction
+author: "Julia Puig"
+cite:
+    authors: "Jingfeng Lu, Fabien Millioz, Damien Garcia, Sebastien Salles, Dong Ye, Denis Friboulet"
+    title:   "Complex Convolutional Neural Networks for Ultrafast Ultrasound Imaging Reconstruction From In-Phase/Quadrature Signal"
+    venue:   "IEEE Transactions on Ultrasonics, Ferroelectrics, and Frequency Control"
+pdf: "https://arxiv.org/pdf/2009.11536.pdf"
+---
+
+
+# Context
+
+* Ultrafast imaging requires less acquisitions than conventional ultrasound focused imaging. However, it needs the **compounding** of several acquisitions to compensate for the lower image contrast and resolution. Therefore, there is a trade-off between image quality and image rate.
+* In a previous work, the same authors developed **ID-Net** (Inception for Diverging-wave Network), a CNN for the compounding of 3 RF acquisitions and reached the same quality than when using 31 RF acquisitions [[1]](https://arxiv.org/abs/1911.03416).
+* The objective of this paper is to develop a CNN that takes IQ signals as an input instead of RF signals, to reduce data size and power requirement. The main difficulty is that, unlike RF signals, **IQ signals are complex**.
+* The authors develop **CID-Net** (Complex-valued Inception for Diverging-wave Network), a complex CNN.
+
+
+# Highlights
+
+* Implementation of a **complex convolution** inspired by the multiplication of complex numbers.
+* Definition of a **complex activation function** based on the complex module.
+* Experimental demonstration that **CID-Net** reaches the same performance than **ID-Net** and outperforms the straightforward method of processing the real and imaginary parts separately (**2BID-Net**, 2-Branch Inception for Diverging-wave Network).
+
+
+# Methods
+
+The complex CNN has 3 main building blocks:
+
+1.  The **complex convolution** was defined in [[2]](https://arxiv.org/abs/1705.09792) and uses complex weights $W = W_r + jW_i$. It is defined as:
+
+$$
+\left[\begin{array}{c}
+\operatorname{Re}(W * X) \\
+\operatorname{Im}(W * X)
+\end{array}\right]=\left[\begin{array}{cc}
+W_{r} & -W_{i} \\
+W_{i} & W_{r}
+\end{array}\right] *\left[\begin{array}{c}
+X_{r} \\
+X_{i}
+\end{array}\right]
+$$
+
+![](/collections/images/complex_cnns/complex_convolution.jpg)
+
+2. The maxout activation function (MU) used in the previous work is replaced by the **amplitude maxout** (AMU). Given a complex convolutional layer output $Z$ and its module $Z_a = |Z|$, the amplitude maxout of $Z$ is defined as:
```

Comment:
$$Z$$ ... $$Z_a = |Z|$$ ... $$Z$$

Reply:
You should put double $$ for each mathematical expression to make them appear latex like in the text ;)